README.rst

```diff
@@ -29,7 +29,7 @@
    :alt: Discourse status
 
 The Python standard library includes a basic
-`SMTP <http://www.faqs.org/rfcs/rfc5321.html>`__ server in the
+`SMTP <https://tools.ietf.org/html/rfc5321>`__ server in the
```

Comment:
`tools.ietf.org` is the authoritative source of RFCs. In addition, using tools.ietf.org linkifies the RFC's TOC + give access to IETF's document tracker, etc.